webpack/assets/javascripts/react_app/components/common/EmptyState/EmptyState.test.js

```diff
@@ -45,25 +45,25 @@ const emptyStatePatternFixtures = {
     icon: 'printer',
     header: 'Printers',
     description: 'Printers print a file from the computer',
-    documentation: 'This is a simple description',
+    documentation: {
+      url: 'https://example.com',
+    },
```

Comment:
can you store documentation as a const in the fixtures and reuse it all over those tests?
actually, bonus points would be to move all of those props into an object in `fixtures.js`, e.g:

```js
export const props = {
  icon,
  header,
  description,
  documentation,
}
```
where this object keys will be also declared in the same file, like `export const icon = 'printer'`.

then in the test it will be just:
```js
'should render main action when given title and url': { ...props, action: actionWithLink }
```